engine/trainer.py

```diff
@@ -634,7 +639,22 @@ def __init__(self, **kwargs):
                             print("Stats with Voronoi")
                             print(r_stats)
                             all_matching_stats_voronoi.append(r_stats)
+                            
+                    if self.cfg.TEST.MATCHING_VJI_PAI and self.cfg.PROBLEM.TYPE == 'INSTANCE_SEG' and self.cfg.DATA.TEST.LOAD_GT:
+                        print("Calculating matching stats using VJI and PAI. . .")
+                        test_file = os.path.join(self.cfg.DATA.TEST.MASK_PATH, filenames[0])
+                        if not os.path.isfile(test_file):
+                            raise ValueError("The mask is supossed to have the same name as the image")
+                        _Y = imread(test_file).squeeze()
+                        r_stats = match_using_VJI_and_PAI(_Y, w_pred)
+                        print(r_stats)
```

Comment:
This value is just an int? If so the best would be to not print it. In the matching stats, the reference you based on, this variable contains not only an int but several metrics with different thresholds as done in Stardist.